src/Elm/Parser/Expression.elm

```diff
@@ -133,21 +133,22 @@ functionCall =
         )
 
 
+glslStart : String
+glslStart =
+    "[glsl|"
+
+
+glslEnd : String
+glslEnd =
+    "|]"
+
+
 glslExpression : Parser State (Node Expression)
 glslExpression =
-    let
-        start : String
-        start =
-            "[glsl|"
-
-        end : String
-        end =
-            "|]"
-    in
     Core.mapChompedString
-        (\s () -> s |> String.dropLeft (String.length start) |> GLSLExpression)
-        (Core.multiComment start end NotNestable)
-        |. Core.symbol end
+        (\s () -> s |> String.dropLeft (String.length glslStart) |> GLSLExpression)
```

Comment:
We should probably also make `String.length glslStart` a constant, right? :thinking: 
Or should/could we get the chomped string on the parser between start and end (without the dropLeft) :thinking: